grafana/base/datasource.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDataSource
+metadata:
+  name: odh-datasources
+spec:
+  name: odh-prometheus.yaml
+  datasources:
+    - name: opendatahub
+      type: prometheus
+      access: proxy
+      url: http://prometheus-operated:9090
```

Comment:
Please follow the  spec from: https://www.redhat.com/en/blog/custom-grafana-dashboards-red-hat-openshift-container-platform-4

```yaml
    - access: proxy
      editable: true
      isDefault: true
      jsonData:
        httpHeaderName1: 'Authorization'
        timeInterval: 5s
        tlsSkipVerify: true
      name: Prometheus
      secureJsonData:
        httpHeaderValue1: 'Bearer ${BEARER_TOKEN}'
      type: prometheus
      url: 'https://thanos-querier.openshift-monitoring.svc.cluster.local:9091'
  name: prometheus-grafanadatasource.yaml
```

We'll overwrite this spec in ksopsed secret in overlay anyways

Reply:
Resolved in latest commit.